pkgs/by-name/ch/chatgpt/update.sh

```diff
@@ -10,10 +10,13 @@ DOWNLOAD_URL=$(echo "$XML_DATA" | xmllint --xpath 'string(//item[1]/enclosure/@u
 
 HASH=$(nix-prefetch-url $DOWNLOAD_URL | xargs nix hash convert --hash-algo sha256)
 
-cat > source.nix << _EOF_
+NIXPKGS_CHATGPT_PATH="$(cd $(dirname ${BASH_SOURCE[0]}) && pwd)"
```

Comment:
```suggestion
NIXPKGS_CHATGPT_PATH="$(dirname ${BASH_SOURCE[0]})"
```
We don't need cd and pwd